### PR TITLE
Update rackspace-metrics-faq.md

### PR DESCRIPTION
**Pull request by joesavak**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-65d59a9b7b/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. My edits are attached. The links to the API doc need to be updated, because we reorganized the content. And then just a few other edits. Let me know if you have any questions. Thanks!

[Rackspace Metrics FAQ_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1156862/Rackspace.Metrics.FAQ_editsKH.pdf)
[Rackspace Metrics FAQ_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1156861/Rackspace.Metrics.FAQ_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5e5ff98de9/how-to/)

File: content/rackspace-metrics/rackspace-metrics-faq.md [hunk 1]
```diff
@@ -1,93 +1,56 @@
...
 permalink: rackspace-metrics-faq/
-audit_date:
+audit_date: '2016-07-19'
 title: Rackspace Metrics FAQ
 type: article
 created_date: '2015-05-06'
 created_by: Constanze Kratel
-last_modified_date: '2016-07-25'
-last_modified_by: Kyle Laffoon
+last_modified_date: '2016-07-19'
```
Comment (catlook) [inline content/rackspace-metrics/rackspace-metrics-faq.md]: errr 2017

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0aefa93297/how-to/)

**Pull request merged**